src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -242,7 +251,7 @@ export interface IProps<OptionType = IDefaultOptionType> extends WithStyles<type
   rowHeight?: number;
   disabled?: boolean;
   isClearable?: boolean;
-  textFieldProps?: { [key: string]: any };
+  textFieldProps?: TextFieldProps;
```

Comment:
As much as I would like this to be `TextFieldProps` I will keep it like this, unless anyone thinks its ok to change.